tests/run-make/libs-through-symlinks/rmake.rs

```diff
@@ -0,0 +1,32 @@
+//! Regression test for [rustc doesn't handle relative symlinks to libraries
+//! #13890](https://github.com/rust-lang/rust/issues/13890).
+//!
+//! The rust compiler searches by default for libraries in its current directory, but used to have
+//! difficulty following symlinks leading to required libraries if the real ones were located
+//! elsewhere. After this was fixed in #13903, this test checks that compilation succeeds through
+//! use of the symlink.
+
+//@ ignore-cross-compile
+
+use run_make_support::{bare_rustc, cwd, path, rfs, rust_lib_name, rustc};
```

Comment:
The last import looks unused?

```suggestion
use run_make_support::{bare_rustc, cwd, path, rfs, rust_lib_name};
```

But it's a good reminder to document the use of `bare_rustc` vs `rustc` as to avoid the implicit `-L` of the latter.

Reply:
Indeed, leftover from trial-and-error.

Reply:
Fixed.